service/worker/replicator/processor.go

```diff
@@ -437,6 +435,22 @@ RetryLoop:
 		}
 		break RetryLoop
 	}
+
+	if retryErr, ok := err.(*shared.RetryTaskError); ok {
```

Comment:
Let's emit a log here with all relevant tags.  This could potentially mean Kafka lost messages.  We should emit a metric on all re-replicate calls initiated by the cluster so we can set up alarms if this starts happening very frequently.

Reply:
we can use the new get raw history API metrics